include/openssl/evp.h

```diff
@@ -1141,6 +1145,8 @@ int EVP_CIPHER_asn1_to_param(EVP_CIPHER_CTX *c, ASN1_TYPE *type);
 /* These are used by EVP_CIPHER methods */
 int EVP_CIPHER_set_asn1_iv(EVP_CIPHER_CTX *c, ASN1_TYPE *type);
 int EVP_CIPHER_get_asn1_iv(EVP_CIPHER_CTX *c, ASN1_TYPE *type);
+int EVP_CIPHER_set_asn1_aead_params(EVP_CIPHER_CTX *c, ASN1_TYPE *type);
+int EVP_CIPHER_get_asn1_aead_params(EVP_CIPHER_CTX *c, ASN1_TYPE *type);
```

Comment:
I think these should probably not be public functions.

Reply:
I thought that they are quite similar to `EVP_CIPHER_set_asn1_iv` resp. `EVP_CIPHER_get_asn1_iv` so I made that public but again happy to make them private.

Reply:
Historically there were lots of functions that were made public that should not have been - so just because there are other similar public functions is probably not a good enough reason to make these ones public. I tend to default to making functions private unless I can see good reasons to make it a public API. Making things public makes it a lot more difficult to refactor code later if we need to. So, I think these should be made private.

Reply:
Ok. Done.